stories/bar/bar.stories.js

```diff
@@ -415,12 +447,13 @@ export const barAsFooter = () => `
 
 barAsFooter.parameters = {
     docs: {
-        iframeHeight: 350
+        iframeHeight: 150,
+        storyDescription: 'The footer bar should display only closing and finalizing action buttons that impact the entire page such as *Save* or *Cancel*. This bar is fixed at the bottom of the screen, and is not visible when scrolling up. It can be displayed by adding the <code>--footer</code> modifier class to the container element.'
```

Comment:
I'm assuming this is fixed at the bottom of the screen, @InnaAtanasova could you confirm if this is true? :)

Reply:
The Bar component has 2 usages - one is as a building part of other components like Dialog, Popover, etc and the other usage is as a building block of a page. You can see the examples [here](https://fundamental-styles.netlify.app/?path=/docs/layouts-page--basic-page). In the first case (as part of other components) it's not fixed at the bottom of the screen, so should remove the `This bar is fixed at the bottom of the screen, and is not visible when scrolling up.` part of the description. It can be added to the [Page](https://fundamental-styles.netlify.app/?path=/docs/layouts-page--basic-page) documentation.

Reply:
Ok, I just removed that sentence from my local repo, but before I reflect changes here... What do you think about adding this sentence to the main component description: **"The bar component is used to construct a Page, and acts as a building block for other components like Dialog, Popover etc."**

Reply:
This is taken from the component wiki `The Bar is a container which is primarily used to hold titles, button and input controls. It consists of three areas left, middle and right. The Bar design and functionality is the basis for Page headers and footers, as well ass toolbars and title bars.`

Reply:
Your sentence also makes sense

Reply:
Did you notice that they also wrote "ass" in the last sentence 🤣

Reply:
Yes :D I laughed so much the first time I saw it :D